launch_param_builder/launch_param_builder.py

```diff
@@ -57,6 +69,9 @@ def yaml(self, file_path: str, parameter_namespace: str = None):
         return self
 
     def file_parameter(self, parameter_name: str, file_path: str):
+        """
+        @deprecated: Use 'yaml' method instead.
```

Comment:
This, I don't know if this method is useful so deprecating but if there are legit usecases I can revert.

Reply:
I don't remember why we added it, I agree that it should be deprecated